src/pods/test-sentences/test-sentences.component.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as styles from 'common/styles/tests.styles';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { Typography, Button } from '@material-ui/core';
+import { SentenceEntityApi } from 'core/sentences';
+import { VerbEntityGlobal } from 'core/verbs';
+
+interface Props {
+  sentenceSelected: SentenceEntityApi;
+  selectedVerb: VerbEntityGlobal;
+  splitSentence: (sentence: string) => string[];
```

Comment:
If this is a business model you don't need to pass this as props.

This is outside from react

Just include it in a business call,import it and directly call it (e.g. when you do a fetch you don't pass it via props)

It's a helper / business method it has nothing to do with React LifeCycle

Reply:
done